### ISSUE TITLE
CT Redesign Search: Cautionary Warnings cannot be expanded or collapsed using a keyboard

### ISSUE THREAD

**Issue by delli-gatti-michael** [department-of-veterans-affairs/va.gov-team#29245]
In the redesigned comparison tool, institutions that have cautionary warnings have a banner on the search result card. This banner can be expanded so that the user can see specifically which warnings apply to the selected institution. Currently, the section can only be expanded or collapsed by clicking and is inaccessible using only a keyboard.

### Steps to Reproduce
1. Navigate to https://staging.va.gov/gi-bill-comparison-tool-sandbox
2. Search for any institution with at least one caution flag (ex. TOP OF THE LINE BARBER COLLEGE)
3. Attempt to expand the cautionary warnings section of the search results card using the keyboard

![Screen Shot 2021-08-25 at 3 05 45 PM](https://user-images.githubusercontent.com/45575003/130851728-ca7de8ca-346a-4187-baeb-6601860874c4.png)

![Screen Shot 2021-08-25 at 3 06 10 PM](https://user-images.githubusercontent.com/45575003/130851755-6cac5f3b-371e-4ef1-af66-ae0b5744dcb7.png)

### Expected Result
The user is able to expand the cautionary warnings section of the search results card using the keyboard

### Actual Result
The user is **not** able to expand the cautionary warnings section of the search results card using the keyboard

**Comment by dmccurdybah** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/18551
